src/main/java/com/superdevs/HealthOMeter/entity/User.java

```diff
@@ -2,7 +2,9 @@
 
 import javax.persistence.*;
 import java.util.Set;
-
+/*
+Contact or User - discussion
```

Comment:
For my perspective both all correct . Only when we decide to use for example User, we need to be Consent from start till end